docs/how_to/install_puppet_enterprise.md

```diff
@@ -0,0 +1,107 @@
+# How-To Install Puppet Enterprise
```

Comment:
maybe `Puppet Enterprise installation reference`? Minor quibble, just not a (hugefan) of `How-To`.